src/LanguageManager/LanguageManagerSettingsPageView.php

```diff
@@ -87,12 +87,14 @@ public function render() {
 
 	private function fire_actions()
 	{
-		$language_id = (string) $this->request->body_value(
+		$language_id = (int) $this->request->body_value(
 			self::QUERY_ARG_ID,
 			INPUT_GET,
 			FILTER_VALIDATE_INT
 		);
 
+		$language_id = $language_id ?? 0;
```

Comment:
Since you did a cast, this will only replace `0` by `0`, and nothing else. I would just remove this line.